http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20220207_autopause_enabled.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="cstrandw" id="autopause_enabled">
+        <addColumn tableName="CLUSTER">
+            <column name="autopauseEnabled" type="boolean" defaultValue="false">
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="set autopauseEnabled to false when autopauseThreshold is 0, else set to true" author="cstrandw">
+        <sql>UPDATE CLUSTER SET autopauseEnabled = TRUE WHERE autopauseThreshold != 0</sql>
```

Comment:
I don't think we need this field. Like I said in slack, `autopauseThreshold` essentially represents all cases already
* None --> autopause disable
* Some(v) --> autopause enabled, and its threshold is “v”

Reply:
I was talking with @gpcarr and @jdcanas about whether we should keep the current implementation or add the enabled value to the database, and we agreed that it would best practice to add it instead of relying on deriving the value every time

Reply:
> deriving the value every time
where will be be "deriving" the value?

The downside of having both `autopauseEnabled` and `autopauseThreshold` is you'll like have invalid states. For instance, you have `autopauseEnabled` being `None`, and `autopauseThreshold` being `Some(v)`....this will be invalid. The two fields aren't independent of each other